Formula/awsume.rb

```diff
@@ -0,0 +1,27 @@
+class Awsume < Formula
+  include Language::Python::Virtualenv
+
+  desc "Utility for easily assuming AWS IAM roles from the command-line"
+  homepage "https://www.trek10.com/blog/awsume-aws-assume-made-awesome"
+
+  url "https://github.com/trek10inc/awsume/archive/4.0.0.tar.gz"
+  sha256 "8cc7d466beef1c32b3397ed62cba22f7a309ab1650317cef6873cf019be7d25e"
+  head "https://github.com/trek10inc/awsume.git", :branch => "master"
```

Comment:
Isn’t `:branch => "master"` already the default?

Reply:
Got it, will remove the redundant info